bears/js/JSStandardBear.py

```diff
@@ -16,13 +17,34 @@ class JSStandardBear:
     """
 
     LANGUAGES = {'JavaScript', 'JSX'}
-    REQUIREMENTS = {NpmRequirement('standard', '10')}
+    REQUIREMENTS = {NpmRequirement('standard', '10'),
+                    NpmRequirement('babel-eslint', '8'),
+                    NpmRequirement('eslint-plugin-flowtype', '2.41.0'),
+                    NpmRequirement('typescript-eslint-parser', '12.0.0'),
+                    NpmRequirement('eslint-plugin-typescript', '0.8.1')}
     AUTHORS = {'The coala developers'}
     AUTHORS_EMAILS = {'coala-devel@googlegroups.com'}
     LICENSE = 'AGPL-3.0'
     CAN_DETECT = {'Syntax'}
     CAN_FIX = {'Formatting'}
     SEE_MORE = 'https://standardjs.com/rules.html'
 
-    def create_arguments(self, filename, file, config_file):
-        return (filename, '--verbose')
+    def create_arguments(self, filename, file, config_file,
+                         globalVars: typed_list(str)=(),
+                         typescript: bool=False,
+                         flow: bool=False):
+        args = ('--verbose', filename,)
+        if flow and typescript:
+            self.err('The javascript variants flow and typescript are ',
+                     'both given but cannot be used together.')
```

Comment:
Could you check if coala doesn't cache results if those two settings conflict, and afterwards they are used correctly?

Reply:
I am not sure how i would do this... do you want me to create a test or check locally on my machine?